apps/arweave/src/ar_nonce_limiter_server.erl

```diff
@@ -12,9 +12,13 @@
 
 -record(state, {}).
 
+%% @doc The number of steps and the corresponding step checkpoints to include in every
+%% regular update.
+-define(REGULAR_UPDATE_INCLUDE_STEPS_COUNT, 2).
```

Comment:
The reason for this is to limit the number of `/vdf3/session` requests that the client makes? i.e. if 1 step gaps are common, then by always sending 2 steps we can elimininate all those /vd3/session queries that we used to do for 1-step gaps

Is that about right? or is there some other reason?

Reply:
Yes, that's the reason